src/plugins/data/common/search/aggs/types.ts

```diff
@@ -156,6 +169,50 @@ export type AggExpressionFunctionArgs<Name extends keyof AggParamsMapping> =
  *
  * @internal
  */
+export interface SerializedAggParamsMapping {
```

Comment:
i can't find this used anywhere but in this same file above ?
- do we need to export it ?
- whats the intention of this change on agg configs ?

Reply:
- You are right, it makes sense not to export this interface.
- `SerializedAggParamsMapping` is representing args, which are passed to the expression functions, but `SchemaConfig` contains params of different types.
Good example: 
https://github.com/elastic/kibana/blob/3494b2af1e961ddcc7233bdd6bb184751325af4d/src/plugins/data/common/search/aggs/metrics/bucket_avg.ts#L18
https://github.com/elastic/kibana/blob/3494b2af1e961ddcc7233bdd6bb184751325af4d/src/plugins/data/common/search/aggs/metrics/bucket_avg.ts#L23

And later usage: 
<img width="499" alt="Screenshot 2022-09-22 at 10 35 43" src="https://user-images.githubusercontent.com/22456368/191686358-b85d5062-6ba5-4c14-9341-8b83027b9b5d.png">
<img width="499" alt="Screenshot 2022-09-22 at 10 35 56" src="https://user-images.githubusercontent.com/22456368/191686390-597b8bc8-56d5-4710-8f8d-209a0d52cb65.png">

We had to add types to the aggParams to make the code more predictable and documented.